selfservice/flow/recovery/handler.go

```diff
@@ -116,9 +116,9 @@ func (h *Handler) RegisterAdminRoutes(admin *x.RouterAdmin) {
 //	Schemes: http, https
 //
 //	Responses:
-//	  200: recoveryFlow
-//	  400: errorGeneric
-//	  default: errorGeneric
+//		200: recoveryFlow
+//		400: errorGeneric
+//		default: errorGeneric
```

Comment:
not sure about this indentation. It doesn't look like the others we have :suspect: 

```suggestion
//    200: recoveryFlow
//    400: errorGeneric
//    default: errorGeneric
```

Reply:
Done, but the formatter turns it into tab + 2 spaces, which is what we use on the other places. But mixing indentation.. 👎🏼